tox.ini

```diff
@@ -42,4 +42,4 @@ deps = mypy==1.3
 # mypy gives different results if you actually install the stuff before you check it
 # don't cache anything, will be slower, but each run should be clean
```

Comment:
Does the comment need to be updated with the latest change?

Reply:
done